hr_payroll/security/hr_payroll_security.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
The model hr.contribution.register has the company_id but I can't see the rule like described [here](https://www.odoo.com/documentation/13.0/howtos/company.html#security-rules). There are more models with these field without the rule

Reply:
thanks, that's a good point, I added those rules now